### ISSUE TITLE
Option to show file at original zoom level (100%) upon opening file, even while in fullscreen or maximized

### ISSUE THREAD

**Issue by BloodRaven0** [jurplel/qView#306]
Currently, no matter which options you select, the default mode is to zoom the image as much as possible in order to fit it in the window size. For those of us that like to keep the viewer maximized at all times, this is really suboptimal, especially when viewing pixel art.
I'm aware that there is a hotkey to view original size, but trying to zoom in or out from that default size blows it back up to fit-in-the-window size, then applies the zoom command. Not to mention it's tedious to have to press a hotkey for each image you open.

**Comment by jurplel** [comment]
Semi-related to https://github.com/jurplel/qView/issues/267
Should be implemented during the refactor of QVGraphicsView and related zooming components

**Comment by jgthms** [comment]
I second this enhancement. I like to browse images in Original Size. Currently, when I use the `O` shortcut, but then it centers on the image, which usually zooms to an irelevant part of the image (since the image is tall). So I have to scroll back to the top. But like you said, this seems related to #267.

A related issue is that the zoom level and the position of the image is not saved, even temporarily. So after having reset the zoom level and moved to the top of the image, I could cycle to the next image and come back, and would have lost my position and zoom level.

So basically, what would be great is:

* have the option to default to "Original Size" when opening a file/navigating through files
* move to the top of the image when going into "Original Size" mode (either through the shortcut or with the aforementioned preference)

Thanks for your work on qView in any case. I just discovered and starting to like it.

**Comment by vassudanagunta** [comment]
Actually you can get this behavior as follows:
1. Disable `Image resizes past actual size`.
2. Size the window to be at least as large as the images you are viewing.

Limitation: When an image is larger than the window size, it will be shrunk to fit.

See also my proposed holistic solution for image and window sizing in #460.

**Comment by jdpurcell** [comment]
Related to #70
